site/en/docs/extensions/mv3/getstarted/index.md

```diff
@@ -354,15 +361,9 @@ Then register the options page in the manifest,
 }
 ```
 
-Reload the extension and click **DETAILS**.
-
-{% Img src="image/BrQidfK9jaQyIHwdw91aVpkPiib2/3fNrYEoJMOPQ00L7tBtp.png",
-       alt="Inspect Views", height="337", width="606" %}
-
-Scroll down the details page and select **Extension options** to view the options page.
+Reload the extension and right-click the extension icon in the toolbar then select **Options**. Alternatively, click DETAILS and scroll down the details page and select **Extension options**.
 
-{% Img src="image/BrQidfK9jaQyIHwdw91aVpkPiib2/z1VEYxYlJev7llaXIQUL.png",
-       alt="Extension Options", height="726", width="645" %}
+{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/aV46PP8KCjEqqenSJxxp.png", alt="Right click to open the options page", width="248", height="260" %}
```

Comment:
This probably needs a new screenshot too; even when the extension is pinned, the puzzle piece should still show.

Reply:
Done!